package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "ecommerce",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@types/node": "20.4.4",
```

Comment:
There's no division between `dependencies` and `devDependencies`? The  template comes like this?

Reply:
Yep, the template comes like that! :akward